bindings/test/unit/test_server.py

```diff
@@ -210,9 +211,9 @@ def test___init___defaults(self):
         self.assertTrue(isinstance(connection.server_wrapper, server.HTTPSServerWrapper))
         self.assertEqual(connection.server_wrapper.pulp_connection, connection)
         self.assertEqual(connection.verify_ssl, True)
-        self.assertEqual(connection.ca_path, server.DEFAULT_CA_PATH)
+        self.assertTrue(connection.ca_path in DEFAULT_CA_PATH_LIST)
         # 1142376 - verify default path points to a known valid file
-        self.assertEqual(server.DEFAULT_CA_PATH, '/etc/pki/tls/certs/ca-bundle.crt')
+        self.assertIsNotNone(server.DEFAULT_CA_PATH)
```

Comment:
This doesn't work in el6 because it's using py2.6's unittest.
`self.assertTrue(server.DEFAULT_CA_PATH is not None)` is probably the most readable alternative.

There's another one over in `common/test/unit/common/test_constants.py` that will need to be fixed.